### ISSUE TITLE
[DOC] Performance note possibly wrong?

### ISSUE THREAD

**Issue by vivi90** [Xinne/osc4py3#12]
## Problem
I am not able to confirm the performance difference between `as_eventloop`, `as_comthreads` and `as_allthreads`, [described in the documentation](https://osc4py3.readthedocs.io/en/latest/userdoc.html#no-thread).

## Results in the documentation
 * `as_eventloop`: fastest
 * `as_comthreads`: medium
 * `as_allthreads`: slowest
 
So it might be good to know, what type of machine is meaned by '[..] my computer [..]'.
Maybe an single core machine? 🤔 
Would explain the results.

## My results
By using [`speedudpcli.py`](https://github.com/Xinne/osc4py3/blob/27e6590c97f18d801afe7b4ac1f5271c426f6696/osc4py3/demos/speedudpcli.py)
```
SCHEDULING: eventloop
...
Elapsed time to send 1000 messages: 0.317 sec in user thread.
...
SCHEDULING: comthreads
...
Elapsed time to send 1000 messages: 0.063 sec in user thread.
...
SCHEDULING: allthreads
...
Elapsed time to send 1000 messages: 0.026 sec in user thread.
...
```
In conclusion (for my machine):
 * `as_eventloop`: slowest
 * `as_comthreads`: medium
 * `as_allthreads`: fastest
 
To be fair i tested the sending alone.
The documentation mentions the [`speedudpsrv.py`](https://github.com/Xinne/osc4py3/blob/27e6590c97f18d801afe7b4ac1f5271c426f6696/osc4py3/demos/speedudpsrv.py) but i can't see any measurement there. 🤔 

## My machine
 * **osc4py3**: `1.0.3`
 * **Python**: `3.10.4`
 * **OS**: Windows 10 Enterprise `Version 21H2 (Build 19044.1826)`
 * **CPU**: AMD Ryzen 7 3700X 3600MHz (8 physical, 16 logical cores)
 * **RAM**: 32GB 3200MHz DDR4
 * **Chipset**: B550 (Socket AM4)
